examples/tutorials/tutorial2-python-ceed.ipynb

```diff
@@ -15,82 +15,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Setting up libCEED for Python\n",
-    "\n",
-    "Clone or download libCEED by running"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "! git clone https://github.com/CEED/libCEED.git"
+    "## Setting up libCEED for Python"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Then move to the libCEED folder"
+    "Install libCEED for Python by running"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "cd libCEED"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "And compile the base library by running"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "! make"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Build libCEED for Python by running"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "python setup.py build install"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Or without superuser permissions"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
We keep getting these small changes in the `execution_count` for various cells. Is there a way to avoid this or somehow reset the notebooks so git is not tracking these meaningless one line changes?

Reply:
In [b46dc66](https://github.com/CEED/libCEED/pull/485/commits/b46dc661041915e76b00551fca48d2d1d7bcff74) I modified a box at the top for the install instruction, and therefore all following boxes have shifted the execution count by one. They are all in order and sequential now (I have personally checked them). I am not aware if there is a way for git to avoid track these particular changes. I didn't think it was a big deal personally.

Reply:
Huh, I was misinterperting the purpose of that value. At some point I thought I saw a high non-sequential number, so I thought that this value was counting the number of times a cell was executed.

Reply:
No, it does! Your interpretation was right. In some of my initial commits I hadn't personally double checked them, and I had committed some that were non sequential. But soon after I realized that and double checked all of them. They are all sequential now. Thanks.